lib/pipeline.js

```diff
@@ -37,28 +35,6 @@ class PipelineModel extends BaseModel {
     }
 
     /**
-     * Format the scm url to include a branch and make case insensitive
-     * @method formatScmUrl
-     * @param  {String}     scmUrl Github scm url
-     * @return {String}            Lowercase scm url with branch name
-     */
-    formatScmUrl(scmUrl) {
```

Comment:
Nevermind, I see that you moved it in your other pull request
https://github.com/screwdriver-cd/screwdriver/pull/88